src/entrypoints/web-vitals.ts

```diff
@@ -1,15 +1,15 @@
-import { onLCP, onINP, onCLS, onFCP } from 'web-vitals'
+import { onLCP, onCLS, onFCP } from 'web-vitals'
+import { onINP } from 'web-vitals/attribution'
 import { assignableWindow } from '../utils/globals'
 
-// TODO export types here as well?
-
 const postHogWebVitalsCallbacks = {
     onLCP,
     onCLS,
     onFCP,
     onINP,
 }
 
-assignableWindow.postHogWebVitalsCallbacks = postHogWebVitalsCallbacks
+assignableWindow.__PosthogExtensions__ = assignableWindow.__PosthogExtensions__ || {}
```

Comment:
every time we add random things onto the window an angel dies (probably)

so I'm slowly moving things onto a single global object on window

this will mean we can start to type that object and have fewer `(window as any)` flying around the place